apps/github-api-integration/behaviours/validate-name.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Model = require('../models/github');
+
+module.exports = config => {
+
+  return superclass => class extends superclass {
+
+    // The process method is part of the POST pipeline. For details
+    // see https://github.com/UKHomeOfficeForms/hof-behaviour-hooks.
+    process(req, res, next) {
```

Comment:
If the thing we are doing is validating then I'd definitely do it on `validate` and not `process`.